### ISSUE TITLE
Can't connect

### ISSUE THREAD

**Issue by anaronix** [3RD-Dimension/3RDD-GCode-Sender-Issues#37]
When trying to connect to GRBL for the first time receive this message:

Method not found: '!!0[] System.Array.Empty()'.

when pressing connect button after first failed attempt receive the message:

Can't connect: Already Connected

Notes:
using the com port that works with bCNC and same baud rate, using controller buffer 127 with Uno
Reset grbl when connecting unchecked (alos tried with checked)

**Comment by 3RD-Dimension** [comment]
What version of GRBL are you using at the moment?

Are you able to post the contents of the log files?  Located in the directory look for Log.txt and Logfile.txt.

**Comment by anaronix** [comment]
Thank you for the reply.
I have resolved the issue. I needed to install .Net 4.6 
I'm not sure if this info is posted on your site, or in the readme and I missed it. I found a reference to the .net requirement in the openCNCpilot info.

Not necessarily relevant now, but here is the info you requested.:

GRBL version 1.1f

Can't see a file called Log.txt, but here is the contents of logfile.txt

`----- Possible levels: Trace, Debug, Info, Warn, Error, Fatal ----|

2019-11-02 10:38:56.0707 | Info  | GCodeSender.MainWindow         | ++++++ 3RDD GCode Sender v2.0.1.2 START ++++++
2019-11-02 10:38:56.4790 | Info  | GCodeSender.MainWindow         | Checking for Updates
2019-11-02 10:38:56.8045 | Error | GCodeSender.MainWindow         | Hotkey file not found, going to create default
2019-11-02 10:38:56.8595 | Info  | GCodeSender.MainWindow         | Creating new HotKey file
2019-11-02 10:38:56.9345 | Info  | GCodeSender.MainWindow         | Hotkey File Created
2019-11-02 10:40:32.4977 | Info  | GCodeSender.MainWindow         | Added Keycode  for KeyFunction JogZPos
2019-11-02 10:40:33.8410 | Info  | GCodeSender.MainWindow         | Added Keycode PageUp for KeyFunction JogZPos
2019-11-02 10:40:34.9980 | Info  | GCodeSender.MainWindow         | Added Keycode  for KeyFunction JogZNeg
2019-11-02 10:40:35.8214 | Info  | GCodeSender.MainWindow         | Added Keycode Next for KeyFunction JogZNeg
2019-11-02 10:40:40.0807 | Info  | GCodeSender.MainWindow         | Added Keycode  for KeyFunction CycleStart
2019-11-02 10:40:42.3492 | Info  | GCodeSender.MainWindow         | Added Keycode Ctrl+S for KeyFunction CycleStart
2019-11-02 10:40:45.1015 | Info  | GCodeSender.MainWindow         | Added Keycode Ctrl+P for KeyFunction FSStop
2019-11-02 10:40:48.0616 | Info  | GCodeSender.MainWindow         | Added Keycode Ctrl+R for KeyFunction RTOrigin
2019-11-02 10:40:53.8042 | Info  | GCodeSender.MainWindow         | Added Keycode Ctrl+Up for KeyFunction FRateInc
2019-11-02 10:40:56.5889 | Info  | GCodeSender.MainWindow         | Added Keycode Ctrl+Down for KeyFunction FRateDec
2019-11-02 10:41:00.6372 | Info  | GCodeSender.MainWindow         | Added Keycode * for KeyFunction JRateInc
2019-11-02 10:41:02.3809 | Info  | GCodeSender.MainWindow         | Added Keycode / for KeyFunction JRateDec
2019-11-02 10:41:09.5815 | Info  | GCodeSender.MainWindow         | Added Keycode S for KeyFunction SpindleOnOff
2019-11-02 10:42:52.3765 | Warn  | GCodeSender.MainWindow         | Method not found: '!!0[] System.Array.Empty()'.`
